client/components/App.jsx

```diff
@@ -16,14 +27,41 @@ class App extends React.Component {
     this.state = {
       images: [],
     };
+    this.fetchImagesByHostelId = this.fetchImagesByHostelId.bind(this);
+  }
+
+  componentDidMount() {
+    // THIS IS HARDCODED CURRENTLY
+    // MUST FIND WAY TO ACCESS CURRENT HOSTEL ID
```

Comment:
We should talk about this at our next stand up as I am currently hard coding a random ID request as well and I'm wondering what everyone else thinks we should be doing since we won't exactly be able to have initial id be by a user interaction.

Other than that, everything else looks good.